ui/index.js

```diff
@@ -1,3 +1,5 @@
+require('react-devtools')
```

Comment:
I think this should be behind some condition but I'm not sure what it is:

```js
if (global.METAMASK_DEBUG) {
  require('react-devtools')
}
```

Here `global.METAMASK_DEBUG` is always falsy for some reason

Reply:
But maybe this doesn't matter as it's not a _huge_ increase to our bundle?

Reply:
I think it should be excluded from production builds. It's big enough.

Using `METAMASK_DEBUG` like you suggested seems perfectly fine to me. Another option would be adding it to the `entries` array in the gulpfile. Using a conditional is probably simpler though - easier to ensure it's only included in the ui bundle this way.

Reply:
Ah sorry, I missed the part about it being undefined. That's because it should be `process.env.METAMASK_DEBUG`.

Reply:
Good catch, I was confused by the use of `global.` a few lines below (which only works because it's _after_ `require('./app/store/store')` which sets it 🤦‍♂